docs/getting-started/running-release.md

```diff
@@ -0,0 +1,143 @@
+# Run bpfd From Release Image
+
+This section describes how to deploy `bpfd` from a given release.
+See [Releases](https://github.com/bpfd-dev/bpfd/releases) for the set of bpfd
+releases.
+
+Jump to the [Setup and Building bpfd](./building-bpfd.md) section
+for help building from the latest code or building from a release branch.
+
+[Tutorial](./tutorial.md) contains more details on the different
+modes to run `bpfd` in on the host and how to test.
+Use [Privileged Mode](#privileged-mode) or [Systemd Service](#systemd-service)
+below for deploying released version of `bpfd` and then use [Tutorial](./tutorial.md)
+for further information on how to test and interact with `bpfd`. 
+
+[Deploying the bpfd-operator](../developer-guide/operator-quick-start.md) contains
+more details on deploying `bpfd` in a Kubernetes deployment and
+[Deploying Example eBPF Programs On Kubernetes](./example-bpf-k8s.md) contains
+more details on interacting with `bpfd` running in a Kubernetes deployment.
+Use [Deploying Release Version of the bpfd-operator](#deploying-release-version-of-the-bpfd-operator)
+below for deploying released version of `bpfd` in Kubernetes and then use the
+links above for further information on how to test and interact with `bpfd`. 
+
+## Privileged Mode
+
+To run `bpfd` in the foreground using `sudo`, download the release binary tar
+files and unpack them.
+
+```console
+export BPFD_REL=0.3.0
+mkdir -p $HOME/src/bpfd-${BPFD_REL}/; cd $HOME/src/bpfd-${BPFD_REL}/
+wget https://github.com/bpfd-dev/bpfd/releases/download/v${BPFD_REL}/bpfctl-linux-x86_64.tar.gz
+wget https://github.com/bpfd-dev/bpfd/releases/download/v${BPFD_REL}/bpfd-linux-x86_64.tar.gz
+
+tar -xzvf bpfctl-linux-x86_64.tar.gz; rm bpfctl-linux-x86_64.tar.gz
+tar -xzvf bpfd-linux-x86_64.tar.gz; rm bpfd-linux-x86_64.tar.gz
+
+$ tree
+.
+├── bpfctl-linux-x86_64.tar.gz
+├── bpfd-linux-x86_64.tar.gz
+└── target
+    └── x86_64-unknown-linux-musl
+        └── release
+            ├── bpfctl
+            └── bpfd
+```
+
+To deploy `bpfd`:
+
+```console
+sudo RUST_LOG=info ./target/x86_64-unknown-linux-musl/release/bpfd 
+[2023-10-13T15:53:25Z INFO  bpfd] Log using env_logger
+[2023-10-13T15:53:25Z INFO  bpfd] Has CAP_BPF: true
+[2023-10-13T15:53:25Z INFO  bpfd] Has CAP_SYS_ADMIN: true
+:
+```
+
+To use `bpfctl`:
+
+```console
+sudo ./target/x86_64-unknown-linux-musl/release/bpfctl list
+ Program ID  Name       Type  Load Time                
+```
+
+Continue in [Tutorial](./tutorial.md) if desired.
+
+## Systemd Service
+
+To run `bpfd` as a systemd service, the binaries will be placed in a well known location
+(`/usr/sbin/.`) and a service configuration file will be added
+(`/usr/lib/systemd/system/bpfd.service`).
```

Comment:
Do we need to add this systemd service file to our release payload?

Reply:
Hrm maybe not your instructions following this seem to cover it by pulling from source code...

But if we want to make it easier to run via systemd we can definitely try to bundle the payload a bit differently.

Reply:
Just to document, we would need (this is without testing, just visual inspection):
./scripts/setup.sh
./scripts/certificates.sh
./scripts/install.sh
./scripts/user.sh
./scripts/bpfd.service

The script also copies the TC, Tracepoint and XDP example bytecode to /run/bpfd/examples/. This would fail and print error, but would complete the install.